nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -135,6 +135,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - The `isabelle` package has been upgraded from 2020 to 2021
 
+- The `openrazer` and `openrazer-daemon` packages as well as the `hardware.openrazer` module now require users to be members of the `openrazer` group instead of `plugdev`. With this change, users no longer need be granted the entire set of `plugdev` group permissions, which can include permissions other than those required by `openrazer`. This can be desirable from a security point of view.
```

Comment:
```suggestion
- The `openrazer` and `openrazer-daemon` packages as well as the `hardware.openrazer` module now require users to be members of the `openrazer` group instead of `plugdev`. With this change, users no longer need be granted the entire set of `plugdev` group permissions, which can include permissions other than those required by `openrazer`. This is desirable from a security point of view.
```